aws/data_source_aws_ec2_instance_type.go

```diff
@@ -0,0 +1,439 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
+)
+
+func dataSourceAwsEc2InstanceType() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsEc2InstanceTypeRead,
+
+		Schema: map[string]*schema.Schema{
+			"instance_type": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"current_generation": {
```

Comment:
Please add a blank between each argument/attribute from lines 16-245.
```suggestion

			"current_generation": {
```